### PR TITLE
Add new modules for OVB analysis, Dressed Hamiltonian and FOBOCI programs. Also, I corrected the CAS+S program

### PR DESCRIPTION
**Pull request by eginer**

### CODE REVIEW

File: plugins/CAS_SD/H_apply.irp.f [hunk 1]
```diff
@@ -20,18 +20,22 @@
 
 
 s = H_apply("CAS_S",do_double_exc=False)
```
Comment (scemama) [inline plugins/CAS_SD/H_apply.irp.f]: `do_double_exc=False` argument is not enough? We have to put  `s.unset_double_excitations()` ?

File: plugins/CAS_SD/cas_sd_selected.irp.f [hunk 2]
```diff
@@ -59,8 +59,9 @@ program full_ci
 
    if(do_pt2_end)then
     print*,'Last iteration only to compute the PT2'
-    threshold_selectors = 1.d0
-    threshold_generators = 0.999d0
+!   threshold_selectors = 1.d0
+!   threshold_generators = 0.999d0
+!   soft_touch threshold_selectors
```
Comment (scemama) [inline plugins/CAS_SD/cas_sd_selected.irp.f]: Are these comments intentional?

File: plugins/FOBOCI/new_approach.irp.f [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+
+
+ deallocate(H_matrix_total)
+ deallocate(psi_good_det)
+ deallocate(dressing_restart_good_det)
+ deallocate(dressing_matrix_restart_1h1p)
+ deallocate(dressing_matrix_restart_2h1p)
+ deallocate(dressing_diag_good_det)
+
+ print*,'YOUPI C EST FINI !!'
```
Comment (scemama) [inline plugins/FOBOCI/new_approach.irp.f]: hmmm......

File: plugins/Hartree_Fock/EZFIO.cfg [hunk 4]
```diff
@@ -33,3 +33,9 @@ doc: If true, skip the (inactive+core) --> (active) and the (active) --> (virtua
 interface: ezfio,provider,ocaml
 default: False
 
+[take_input_guess]
```
Comment (scemama) [inline plugins/Hartree_Fock/EZFIO.cfg]: If you don't want to use the current MOs, you should delete them by hand. It is too complicated.

Comment (eginer) [inline plugins/Hartree_Fock/EZFIO.cfg]: You are right, actually I do not make any use of that ... 
I canceled it !

**Pull request merged**